indico/web/templates/forms/dropzone_widget.html

```diff
@@ -15,7 +15,7 @@
             </div>
         </div>
     </div>
-    <input class="change-trigger" type="hidden" value="0" name="__file_change_trigger">
+    <input class="change-trigger" type="hidden" value="{{ 1 if field.data else 0}}" name="__file_change_trigger">
```

Comment:
ok, sounds good. i think the problem with the very first code was that we were using the file count, so replacing a file didn't change it